engine/engine/content/builtins/input/default.gamepads

```diff
@@ -5367,43 +5409,11 @@ driver
     map { input: GAMEPAD_GUIDE type: GAMEPAD_TYPE_BUTTON index: 8 }
 }
 
-driver
```

Comment:
This was a left-over attempt to have a generic Android controller. It should not have been included in the original PR for Android controller support